src/Microsoft.AspNetCore.SignalR.Client/RpcConnection.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Pipelines;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public class RpcConnection : IDisposable
```

Comment:
Should this be called HubConnection? (I am planning to rename the RpcConnection in ts client to HubConnection because we decided not to expose RPC per se).